src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/nonassociations/NonAssociationsMigrationServiceTest.kt

```diff
@@ -978,6 +978,28 @@ internal class NonAssociationsMigrationServiceTest {
       )
     }
 
+    @Test
+    internal fun `will not migrate non-primary associations`(): Unit = runBlocking {
```

Comment:
think we're trying to use `runTest` instead of `runBlocking` as it skips any delays (but not sure there will be any difference here)